pkg/config/config.go

```diff
@@ -99,6 +101,12 @@ func (cfg *Config) BindFlags() {
 		[]string{},
 		"Configures the ACK service controller to always set key/value pairs tags on resources that it manages.",
 	)
+	flag.StringVar(
+		&cfg.ListenNamespace, flagListenNamespace,
+		"",
+		"Specific namespace the service controller will listen to for object creation from CRD. "+ 
+			" By default it will listen to all namespaces",
```

Comment:
you can also add that in the default case, the controller will ignore the `kube-system`, `kube-public` and `ack-system` namespaces

Reply:
Nice catch but actually the controller will listen to all the namespaces by default 
only the namespace cache will ignore kube-system and kube-public 

added comment to `namespace cache`